libs/openFrameworks/sound/ofRtAudioSoundStream.h

```diff
@@ -5,6 +5,7 @@
 #include "ofBaseSoundStream.h"
 #include "ofTypes.h"
 #include "ofSoundBuffer.h"
+#include "RtAudio.h"
 
 class RtAudio;
```

Comment:
ah my bad I can remove that :laughing: my multitasking during class is atrocious